### ISSUE TITLE
Hide tab text when using vertical tabs - autohide_tabstoolbar_v2.css 

### ISSUE THREAD

**Issue by Stoarr** [MrOtherGuy/firefox-csshacks#476]
Is it possible to hide the tab text when vertical tabs are collapsed with using autohide_tabstoolbar_v2.css? Basically have it look like the standard collapsed vertical tabs? Not really a fan of having the first letter of the tab shown.
Also, Is there a way to have the auto-hide feature while simultaneously having a method (button, hotkey etc.) to keep the expanded tabs open?

**Comment by MrOtherGuy** [comment]
Yeah, it should be pretty simple to hide just the text. But if you were to change the layout as well then things could become a bit more complicated - that is if you intend to also make the size of tabs narrow.

To be able to toggle to keep the expanded state you would need to do something more than what can be achieved with css alone. You could for example write an extension that assigned some prefix to window title, and then have extra style rules that would conditionally apply based on that title prefix. It will totally work but you need a custom extension for that.
